packages/lib/gpc/test/gpc.spec.ts

```diff
@@ -204,6 +281,86 @@ describe("gpc library (Precompiled Artifacts) should work", async function () {
         semaphoreV3: ownerIdentity,
         externalNullifier
       },
+      membershipLists: {
+        list1: [[sampleEntries.G, sampleEntries2.ticketID]].concat(
+          [
+            [87, 1],
+            [99, 8],
+            [8273, 0],
+            [0, 0],
+            [12387, 3],
+            [99999, 66],
+            [653, 362374823],
+            [29387, 1236478236],
+            [1238, 9238374],
+            [1, 1],
+            [87, 87]
+          ].map((pair) => [
+            { type: "int", value: BigInt(pair[0]) },
+            { type: "cryptographic", value: BigInt(pair[1]) }
+          ])
+        ),
+        list2: [
+          [129384723n, 123746238746n, 1237n, 18239n],
+          [1283748973n, 0n, 1n, 2n],
+          [9023874n, 8237n, 23674n, 23874n]
+        ]
+          .map(
+            (tuple) =>
+              [
+                { type: "cryptographic", value: tuple[0] },
+                { type: "int", value: tuple[1] },
+                { type: "int", value: tuple[2] },
+                { type: "cryptographic", value: tuple[3] }
+              ] as PODValueTuple
+          )
+          .concat([
+            [
+              sampleEntries2.ticketID,
+              sampleEntries.otherTicketID,
+              sampleEntries.G,
+              sampleEntries.owner
+            ]
+          ]),
+        goats: [
+          0, 7, 87, 11, 2938, 1923483, 123948, 12839428374, 1234,
+          12343487239487, 2, 3
+        ]
+          .map(
+            (value: number) =>
+              [
+                {
+                  type: "cryptographic",
+                  value: BigInt(value)
+                } as PODCryptographicValue
+              ][0]
```

Comment:
What's the reason for this `[x][0]` form vs. `x`?

Reply:
JavaScript's syntactic ambiguity when it comes to blocks and objects. The alternative would be a block with a return statement.

Reply:
Oh, I'd find the return easier to understand, though I don't care strongly if this is a common trick for JS/TS programmers.